utils.py

```diff
@@ -1108,7 +1108,7 @@ def norm(x, encoding="latin1"):
 def date(fmt=None, timestamp=None):
     "Manejo de fechas (simil PHP)"
     if fmt == "U":  # return timestamp
-        t = datetime.datetime.now()
+        t = datetime.datetime.utcnow()
```

Comment:
Idem, if this change is not related it would be better to be in a separate PR
Also please add a comment in the code or commit description to understand why this change is needed
I remember something about the Argentina vs India timezone, for example you can add 
```suggestion
        # use universal standard time to avoid timezone differences
        t = datetime.datetime.utcnow()
```